src/main/java/ch/njol/skript/expressions/ExprExperienceCooldown.java

```diff
@@ -0,0 +1,92 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Experience Cooldown")
+@Description("The experience cooldown of a player")
+@Examples({
+	"send experience cooldown of player",
+	"set experience cooldown of player to 1 hour",
+	"if experience cooldown of player >= 10 minutes:",
+		"\tclear experience cooldown of player"
+})
+@Since("INSERT VERSION")
+public class ExprExperienceCooldown extends SimplePropertyExpression<Player, Timespan> {
+
+	static {
+		register(ExprExperienceCooldown.class, Timespan.class, "[the] (experience|exp|xp) cooldown", "players");
```

Comment:
```suggestion
		register(ExprExperienceCooldown.class, Timespan.class, "[the] (experience|[e]xp) cooldown", "players");
```
This is mainly just a preference thing, up to you. If you decide you like it, this can be changed elsewhere too